packages/common/core-interfaces/src/exposedUtilityTypes.ts

```diff
@@ -0,0 +1,565 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable @rushstack/no-new-null */
+
+import type {
+	SerializationErrorPerNonPublicProperties,
+	SerializationErrorPerUndefinedArrayElement,
+} from "./jsonSerializationErrors.js";
+import type { JsonTypeWith, NonNullJsonObjectWith } from "./jsonType.js";
+
+/**
+ * Collection of utility types that are not intended to be used/imported
+ * directly outside of this package.
```

Comment:
Might be worth including in this comment they may not follow semver, since I assume thats the intention (both to communicate it to devs working on/using/reviewing these types but also to customers)

Reply:
I think the `@system` tag is now sufficiently documented to convey that. LMK if you think otherwise.